shared/engine/transport-shared.tsx

```diff
@@ -1,53 +1,14 @@
 // Classes used to handle RPCs. Ability to inject delays into calls to/from server
-import rpc from 'framed-msgpack-rpc'
+import * as Framed from 'framed-msgpack-rpc'
 import {printRPC, printRPCWaitingSession} from '../local-debug'
 import {requestIdleCallback} from '../util/idle-callback'
 import * as LocalConsole from './local-console'
 import * as Stats from './stats'
 
-const RobustTransport = rpc.transport.RobustTransport
-const RpcClient = rpc.client.Client
-
-rpc.pack.set_opt('encode_lib', '@msgpack/msgpack')
-
-// We basically always log/ensure once all the calls back and forth
-function _wrap(options: {
```

Comment:
this wrapper was always confusing to me (i wrote it) whenever i revisited it and it makes the typing / debugging much harder to deal with so just remove it